demos/unity_states/cs/.gitignore

```diff
@@ -1,6 +1,7 @@
 /src/bin/
 /src/obj
-/src/generated/
+
+/generated/
```

Comment:
Generated files from the `build_mac_host` script are placed in `/src/generated`. What is the purpose of this change?

Reply:
on my machine they were placed in the changed folder. Which is the correct location?

Reply:
`/src/generated` is the correct location as executed in the build script. Can you post a screenshot of the file tree in Finder of `/generated` (or the output of `tree`?

Reply:
resolved. Not sure why I thought it needed changing.